metricbeat/docs/fields.asciidoc

```diff
@@ -1261,7 +1284,13 @@ timecheck round
 *`ceph.cluster_health.timechecks.round.status`*::
 +
 --
+<<<<<<< HEAD
```

Comment:
it seems some merge errors happened, you will need to run make update

Reply:
yep, currently waiting for make update to finish 👍